src/.fuse_hidden0000271c00000002

```diff
@@ -0,0 +1 @@
+/* You can add global styles to this file, and also import other style files */
```

Comment:
I don't think that this file is needed.

Reply:
This file was added by default, I've even removed it.